src/components/ProjectToolBarSection/ProjectToolBarSection.jsx

```
@@ -76,13 +79,20 @@ const SortByLabelWrapper = styled.label`
+  height: ${theme.typography.mediumIconSize};
+`
+
 const ProjectToolBarSection = ({
   projectFilter,
   setProjectFilter,
   projectSortKey,
   setProjectSortKey,
   setIsProjectSortAsc,
   addProjectToProjectsPage,
+  isExploreLaunchEnabledForUser,
+  handleExploreButtonClick,
```

ESLint

[error] 'handleExploreButtonClick' is missing in props validation

(react/prop-types)